data/json/mapgen_palettes/acidia_camp_palette.json

```diff
@@ -84,7 +84,8 @@
       "u": "f_cupboard",
       "W": "f_canvas_wall",
       "w": "f_null",
-      "y": "f_smoking_rack"
+      "y": "f_smoking_rack",
+      "F": "f_arcfurnace_empty"
```

Comment:
duplicate because of `"F": "f_fireplace"` further up

Reply:
P.S. Arc furnace PR was closed a while ago?

Reply:
Was it a stand alone arc furnace?  This is part of the needed stuff to make acetylene.